ds/beis_indicators/utils/dir_file_management.py

```diff
@@ -0,0 +1,93 @@
+#Utilities to manage directories and files including saves, creation of schemas etc.
+
+import os
+import pandas as pd 
+import re
+import beis_indicators
+
+project_dir = beis_indicators.project_dir
+
+def make_dirs(name,dirs = ['raw','processed']):
+    '''
+    Utility that creates directories to save the data
+    
+    '''
+    
+    for d in dirs:
+        if name not in os.listdir(f'{project_dir}/data/{d}'):
+            os.mkdir(f'{project_dir}/data/{d}/{name}')
+
+def tidy_cols(my_csv):
+    '''
+    Tidies column names ie lower and replace spaces with underscores
+    
+    '''
+    
+    return([re.sub(' ','_',col.lower()) for col in my_csv.columns])
+
+
+def get_nuts_category(year):
+    '''
+    Function that returns the nuts2 year in place for a year
+    '''
+    for t in [2016,2013,2010,2006,2003]:
+        if year >=t:
+            return(f'nuts2_{str(t)}')
+
+def get_nuts_spec(year):
+    '''
+    Function that returns the nuts2 year in place for a year
+    '''
+    for t in [2016,2013,2010,2006,2003]:
```

Comment:
I'd store this array `[2016,2013,2010,2006,2003]` in a var in some utils module (maybe in `aux/`?) and I'd import it so that it can be shared by all scripts

Reply:
The function itself is in a package in the `utils` module so sharable across scripts. I will keep it as it for now.